### PR TITLE
docs: Update DEV_ENV.md

### PR DESCRIPTION
**Pull request by dimple-jethani**
Updating DEV_END.md with docker info.

### CODE REVIEW

**Review by tihuan — APPROVED** [review]
LGTM! Thanks so much for updating the doc, @dimple-jethani 🙏 !! 

I think we just need to change the commit message format to follow conventional commits!

In this case, probably just `docs: Add Docker Desktop` 🙆‍♂️ 

https://github.com/pvdlg/conventional-commit-types#commit-types

File: DEV_ENV.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 ## Development quickstart
 
-1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`.
-1. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
-1. From the root of this repository, run `make local-init` to build and run the dev environment. The first build takes awhile, but subsequent runs will use cached artifacts.
-1. Visit [http://backend.corporanet.local:5000](http://backend.corporanet.local:5000) to view the backend, and [http://frontend.corporanet.local:3000](http://frontend:3000) for the frontend.
-1. `make local-dbconsole` starts a connection with the local postgresql db.
-1. **Open the source code and start editing!**
+1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`. If you have a Mac, [install Docker Desktop](https://www.docker.com/products/docker-desktop) and open it so it's running on your machine.
+2. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
```
Comment (MDunitz) [inline DEV_ENV.md:6]: I think if you leave theses all as 1s markdown will auto convert them to the correct numbers and then you dont need to worry about renumbering if you add a new step

**Review by seve — APPROVED** [review]

File: DEV_ENV.md (hunk 1 above)
Comment (maniarathi) [reply DEV_ENV.md:6]:
```suggestion
1. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
```

File: DEV_ENV.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
+3. From the root of this repository, run `make local-init` to build and run the dev environment. The first build takes awhile, but subsequent runs will use cached artifacts.
```
Comment (maniarathi) [inline DEV_ENV.md:7]:
```suggestion
1. From the root of this repository, run `make local-init` to build and run the dev environment. The first build takes awhile, but subsequent runs will use cached artifacts.
```

File: DEV_ENV.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
+4. Visit [http://backend.corporanet.local:5000](http://backend.corporanet.local:5000) to view the backend, and [http://frontend.corporanet.local:3000](http://frontend:3000) for the frontend.
```
Comment (maniarathi) [inline DEV_ENV.md:8]:
```suggestion
1. Visit [http://backend.corporanet.local:5000](http://backend.corporanet.local:5000) to view the backend, and [http://frontend.corporanet.local:3000](http://frontend:3000) for the frontend.
```

File: DEV_ENV.md [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
+5. `make local-dbconsole` starts a connection with the local postgresql db.
```
Comment (maniarathi) [inline DEV_ENV.md:9]:
```suggestion
1. `make local-dbconsole` starts a connection with the local postgresql db.
```

File: DEV_ENV.md [hunk 5]
```diff
@@ -2,12 +2,12 @@
...
+6. **Open the source code and start editing!**
```
Comment (maniarathi) [inline DEV_ENV.md:10]:
```suggestion
1. **Open the source code and start editing!**
```

**Pull request merged**